g3doc/quick_reference.md

```diff
@@ -412,6 +412,26 @@ example `Lt` instead of `operator<`.
     lane has index 0. This is useful in tests for detecting lane-crossing bugs.
 *   <code>V **SignBit**(D, T)</code>: returns N-lane vector with all lanes set
     to a value whose representation has only the most-significant bit set.
+*   `D`: `{u,i}8` \
+    <code>V **Dup128VecFromValues**(D d, T t0, T t1, T t2, T t3, T t4, T t5,
+    T t6, T t7, T t8, T t9, T t10, T t11, T t12, T t13, T t14, T t15)</code>:
+    Creates a vector with the following values, broadcasted to each 16-byte
```

Comment:
Would you mind updating the documentation so that there is a single op, for all types?
We could express the varying argument count as T t0, .., T tK where K is 16/sizeof(T) - 1.

Would also be good to explain how replication works: t0 goes into the least-significant lane of each 128-bit block.

Reply:
Updated Dup128VecFromValues documentation